### DISCUSSION TITLE
N ways to use the HTML API in WordPress

### DISCUSSION THREAD

**Discussion by ndiego** [WordPress/developer-blog-content#313] [Topic Ideas]
The title is TBD, but the idea of this article is to showcase the many different ways that you can use the [HTML API](https://make.wordpress.org/core/2023/03/07/introducing-the-html-api-in-wordpress-6-2/) in WordPress plugins and themes. The article will begin with an overview of the HTML API followed by N practical examples that developers can use in their own projects. 

The HTML API is an incredibly useful new addition to WordPress, but we have not showcased it enough on the blog. The goal of this article is to inspire folks to begin using the API and exploring what it can do. The article should also include a quick highlight of future enhancements that will be coming in future WordPress releases.

**Comment by justintadlock** [comment]
Noting that I'm up for closing https://github.com/WordPress/developer-blog-content/discussions/301 in favor of this topic that gives us some more foundational examples of using the HTML API first.

**Reply by ndiego** [reply]
Works for me!

**Comment by bph** [comment]
the topic was approved. The conversation continues on the issue: 
- #451
